jwst/tweakreg/__init__.py

```diff
@@ -7,8 +7,8 @@
 __docformat__ = 'restructuredtext'
 
 __taskname__ = 'tweakreg'
-__version__ = '0.8.1'
-__vdate__ = '14-November-2017'
+__version__ = '0.8.2'
```

Comment:
We should bump this up to 0.9.1, to keep in synch with the current development work that will be released as 0.9.x

Reply:
OK